### ISSUE TITLE
Anomalous versions

### ISSUE THREAD

**Issue by elazar** [vinkla/climb#16]
```
opauth/opauth                      1.0.x-dev     →     1.0.0-alp
```

The available version string is getting truncated: the full string is [1.0.0-alpha.1](https://packagist.org/packages/opauth/opauth#1.0.0-alpha.1).

```
league/factory-muffin              2.1.1         →     3.0.0
```

The available version string implies a stable release, where the actual version string is [v3.0.0-RC1](https://packagist.org/packages/league/factory-muffin#v3.0.0-RC1). This makes the output less useful, as I have to go to Packagist to confirm whether or not the release is stable.

**Comment by vinkla** [comment]
Yeah, we should exclude `RC` versions. Thanks for reporting!

**Comment by vinkla** [comment]
This should be fixed in 0.4.0
